library/OrderCloud.Catalyst/Integrations/Implementations/Vertex/Models/VertexException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OrderCloud.Catalyst;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OrderCloud.Catalyst
+{
+	public class VertexException : CatalystBaseException
+	{
+		public VertexException(List<VertexResponseError> errors) : 
+			base(
+				"VertexTaxCalculationError", 
+				"The vertex api returned an error: https://restconnect.vertexsmb.com/vertex-restapi/v1/sale", 
+				errors, 
+				int.Parse(errors.First().status
```

Comment:
What do you think of using the `FirstOrDefault` method instead? It will avoid exception if for any reason the `errors` is empty.

Reply:
Good call, Updated!